src/main/java/sirius/web/services/JSONStructuredOutput.java

```diff
@@ -244,7 +243,7 @@ private void writePlainProperty(String name, Object data) {
             } else if (data instanceof Boolean || data instanceof Number) {
                 writer.write(data.toString());
             } else if (data instanceof Amount) {
-                writer.write(((Amount)data).toMachineString());
+                writer.write(((Amount) data).toMachineString());
```

Comment:
Question: Is the cast necessary any longer with Java 16 pattern matching?

Reply:
yep (tried to remove it)

Reply:
I think with `data instanceof Amount amount` you can avoid the cast

Reply:
Ahhh yes, thats how the pattern matching is supposed to work. So you need to specify the variable name of the cast instance

Reply:
niiiice tipp, thx